src/appmixer/microsoft/mail/CreateDraft/component.json

```diff
@@ -0,0 +1,90 @@
+{
+    "name": "appmixer.microsoft.mail.CreateDraft",
+    "author": "Appmixer <info@appmixer.com>",
+    "description": "Create a draft email.",
+    "auth": {
+        "service": "appmixer:microsoft",
+        "scope": ["Mail.ReadWrite"]
+    },
+    "inPorts": [
+        {
+            "name": "in",
+            "inspector": {
+                "inputs": {
```

Comment:
there are more in properties in the `message` type [spescs](https://learn.microsoft.com/en-us/graph/api/resources/message?view=graph-rest-1.0#properties). Can we add more? at least bcc (comparing with gmail.CreateDraft)